karrot/groups/api.py

```diff
@@ -201,6 +201,30 @@ def trust_user(self, request, pk, user_id):
 
         return Response(data={})
 
+    @action(
+        detail=True,
+        methods=['POST'],
```

Comment:
As mentioned in the frontend PR, using `DELETE` method on the same endpoint as the trust one would fit in better with what we've done so far.

Reply:
done